api/requirements-test.txt

```diff
@@ -1,7 +1,9 @@
 anyio==3.6.2
+azure-storage-blob==12.16.0
```

Comment:
do we need to keep 2 sets of requirements files?

Reply:
The test version originally differed with the pymongo in-memory db for testing. However, it now features in the requirements.txt too now. I will delete the -test one.